x-pack/test/security_solution_endpoint/apps/endpoint/policy_details.ts

```diff
@@ -199,15 +199,39 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
               },
               policy: {
                 linux: {
+                  advanced: {
```

Comment:
@kevinlog should i remove these as well? i'm not sure what purpose the default values in this file are serving

Reply:
yes, these tests will fail in the short term before we actually test adding advanced configs since they're not in the defaults

Reply:
23238e9d29504f34a9e774cad99443b830951c52